governance/program/src/processor/mod.rs

```diff
@@ -0,0 +1,58 @@
+//! Program processor
+
+pub mod process_create_realm;
+pub mod process_deposit_governing_tokens;
+pub mod process_set_vote_authority;
+pub mod process_withdraw_governing_tokens;
+
+use crate::instruction::GovernanceInstruction;
+use borsh::BorshDeserialize;
+
+use process_create_realm::process_create_realm;
```

Comment:
How about `use process_create_realm::*`: `use foo::foo` is pretty tedious.   Same for all others here.  A single processor.rs file would suite me fine too

Reply:
Done. I stayed with an instruction per module/file